<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>80</y>
      <width>511</width>
      <height>371</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Дата</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Следующая дата</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Разница дней</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="openFileButton">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>90</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Открыть файл</string>
    </property>
   </widget>
   <widget class="QLabel" name="currentDateLabel">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>20</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="indexLineEdit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>490</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="showPreviousDayButton">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>150</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Предыдущий день</string>
    </property>
   </widget>
   <widget class="QPushButton" name="showWeekNumberButton">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>210</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Номер недели</string>
    </property>
   </widget>
   <widget class="QPushButton" name="showDurationButton">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>270</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Разница в днях</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="BirthdayDateLineEdit">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>490</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="IndexOfDateLabel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>460</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Номер Даты</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="BirthdayDateLabel">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>460</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Дата Дня Рождения</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="showDaysTillBdButton">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>330</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Дней до ДР</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AddDateInFileButton">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>390</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить в файл</string>
    </property>
   </widget>
   <widget class="QPushButton" name="UpdateDateButton">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>470</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Изменить  дату</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>37</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
